app/assets/javascripts/application.js

```diff
@@ -6,5 +6,7 @@
 //= require analytics-enhancedEcommerce
 //= require cookies
 window.CookieSettings.start()
-window.GOVUK.analyticsInit()
 window.GOVUKFrontend.initAll()
+if (window.GOVUK.analyticsInit) {
```

Comment:
Can we move this if statement inside the `analyticsInit()` function please? That way we keep the `application.js` free of any logic.